### PR TITLE
Update enter-ccupdate.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the description

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enter-ccupdate.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.topic: conceptual
 ms.prod: skype-for-business-itpro
 localization_priority: Normal
 ms.assetid: 330367f2-22b0-43e3-b8fb-3e0d2e3b330e
-description: "The Enter-CcUpdate cmdlet prepares the Skype for Business Cloud Connector Edition host server for the update process by putting it in maintenance mode. The appliance isdrained—that is, all existing calls will complete, but new calls are rejected."
+description: "The Enter-CcUpdate cmdlet prepares the Skype for Business Cloud Connector Edition host server for the update process by putting it in maintenance mode. The appliance immediately stops all services ending any ongoing calls and will reject any new calls"
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enter-ccupdate.md]:
```suggestion
description: "The Enter-CcUpdate cmdlet prepares the Skype for Business Cloud Connector Edition host server for the update process by putting it in maintenance mode. The appliance immediately stops all services, ending any ongoing calls and rejecting any new calls"
```

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enter-ccupdate.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ ms.topic: conceptual
 ms.prod: skype-for-business-itpro
 localization_priority: Normal
 ms.assetid: 330367f2-22b0-43e3-b8fb-3e0d2e3b330e
-description: "The Enter-CcUpdate cmdlet prepares the Skype for Business Cloud Connector Edition host server for the update process by putting it in maintenance mode. The appliance isdrained—that is, all existing calls will complete, but new calls are rejected."
+description: "The Enter-CcUpdate cmdlet prepares the Skype for Business Cloud Connector Edition host server for the update process by putting it in maintenance mode. The appliance immediately stops all services, ending any ongoing calls and rejecting any new calls"
```
Comment (nenonix) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/enter-ccupdate.md]:
```suggestion
description: "The Enter-CcUpdate cmdlet prepares the Skype for Business Cloud Connector Edition host server for the update process by putting it in maintenance mode. The appliance immediately stops all services, ending any ongoing calls and rejecting any new calls."
```

**Comment by nenonix** [comment]
Did you only mean to change the metadata description? The same content repeats on line 18

**Comment by msbemba** [comment]
> Did you only mean to change the metadata description? The same content repeats on line 18

I wasnt able to add a new comit So created another PR https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/2352#issuecomment-497571688

**Comment by scanum** [comment]
@CarolynRowe Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**